### ISSUE TITLE
[sentry-vite-debug-id-upload-plugin] Client network socket disconnected before TLS connection was established

### ISSUE THREAD

**Issue by dante01yoon** [getsentry/sentry-javascript-bundler-plugins#463]
### Environment
node 16.13.1
What version are you running? Etc.

 "@sentry/react": "7.75.1",
 "@sentry/vite-plugin": "2.9.0",
 "vite": "^4.4.5",

npm 
### Steps to Reproduce

1. upload sentry sourcemap after build
2. this is how my sentryVitePlugin specified in `vite.config.ts`
```ts


export default defineConfig(({ mode }) => {

return {
  // ...
plugins: [
react(),
sentryVitePlugin({
      org: "my-sentry",
       project: "my-sentry-web-project",
       authToken: process.env.SENTRY_AUTH_TOKEN,
       errorHandler: function(err) {
       console.log("sentryVitePlugn err during upload: ", err), 
})
],
 // ... 
}
...
```
it worked a week ago, but broken recently. 

### Expected Result
should be build successfully 

### Actual Result
![스크린샷 2024-01-18 오후 2 54 30](https://github.com/getsentry/sentry-javascript-bundler-plugins/assets/6510430/dbb27375-fb4a-4c74-ab20-379dd712c66c)

What actually happened. Maybe a screenshot/recording? Maybe some logs?

**Comment by lforst** [comment]
Hi, thanks for reporting this!

- Are you on SaaS or self hosted Sentry?
- Is this consistently happening?
- Can you try setting `telemetry: false` in the options and see if it is still happening? Thanks!

**Comment by dante01yoon** [comment]
It's not happening with `telemetry: false`, however, I'm assume this should be work without set false to telemetry.

**Comment by lforst** [comment]
Yeah correct. We changed something a few days ago, lemme release that an we'll try again!

**Comment by dante01yoon** [comment]
> @dante01yoon We just released version `2.10.3`. Can you try that one out? Thanks!

It works with telemetry flag. Thanks for quick catching up!

**Comment by dante01yoon** [comment]
Resolved  via `2.10.3` update
